src/test/java/org/takes/rs/BodyByteArrayTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.rs;
+
+import org.cactoos.io.BytesOf;
+import org.cactoos.io.LengthOf;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.Test;
+import org.takes.misc.Utf8String;
+
+/**
+ * Test case for {@link Body.ByteArray}.
+ *
+ * @since 1.15
+ */
+public final class BodyByteArrayTest {
+
+    /**
+     * Body.ByteArray can provide the expected input.
+     * @throws Exception If some problem inside.
+     */
+    @Test
+    public void returnsCorrectInputWithByteArray() throws Exception {
+        final byte[] bytes =
+            new Utf8String("ByteArray returnsCorrectInput!").asBytes();
+        MatcherAssert.assertThat(
+            "ByteArray doesn't provide correct bytes",
```

Comment:
@Serranya Please, add the name "body content" into the message to be more clear and specific. It's too much generic.